hs_core/views/utils.py

```diff
@@ -34,7 +43,18 @@ def validate_json(js):
     try:
         json.loads(js)
     except ValueError:
-        raise exceptions.ValidationError('Invalid JSON')
+        raise ValidationError(detail='Invalid JSON')
+
+
+def validate_user_name(user_name):
+    if not User.objects.filter(username=user_name).exists():
+        raise ValidationError(detail='No user found for user name:%s' % user_name)
+
+
+def validate_group_name(group_name):
+    if not Group.objects.filter(name=group_name).exists():
+        raise ValidationError(detail='No group found for group name:%s' % group_name)
```

Comment:
We only have as single group, so should the group even appear in the API at all?